packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -15,15 +15,15 @@ URL: http://www.rabbitmq.com/
 BuildArch: noarch
 BuildRequires: erlang >= %{erlang_minver}, python-simplejson, xmlto, libxslt, gzip, sed, zip, rsync
 
-%if 0%{?fedora} || 0%{?rhel} >= 7 || 0%{?suse_version}
+%if 0%{?fedora} || 0%{?rhel} >= 7 || 0%{?suse_version} >= 1315
```

Comment:
This constant is described on the [openSUSE cross-distribution build](https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto) page.